validation-test/SIL/verify_all_overlays.py

```diff
@@ -39,7 +39,7 @@
 
     # llvm-bcanalyzer | not grep Unknown
     bcanalyzer_output = subprocess.check_output(["llvm-bcanalyzer",
-                                                 module_path])
+                                                 module_path]).decode("utf-8")
```

Comment:
Ugh, this is what we get for not having the validation tests fully passing on Windows yet.  Thank you!